src/Components/Aspire.RabbitMQ.Client/AspireRabbitMQExtensions.cs

```diff
@@ -48,18 +48,20 @@ public static class AspireRabbitMQExtensions
     /// </summary>
     /// <param name="builder">The <see cref="IHostApplicationBuilder" /> to read config from and add services to.</param>
     /// <param name="name">The name of the component, which is used as the <see cref="ServiceDescriptor.ServiceKey"/> of the service and also to retrieve the connection string from the ConnectionStrings configuration section.</param>
+    /// <param name="connectionName">A name used to retrieve the connection string from the ConnectionStrings configuration section. If not specified, the <paramref name="name"/> will be used.</param>
     /// <param name="configureSettings">An optional method that can be used for customizing the <see cref="RabbitMQClientSettings"/>. It's invoked after the settings are read from the configuration.</param>
     /// <param name="configureConnectionFactory">An optional method that can be used for customizing the <see cref="ConnectionFactory"/>. It's invoked after the options are read from the configuration.</param>
     /// <remarks>Reads the configuration from "Aspire:RabbitMQ:Client:{name}" section.</remarks>
     public static void AddKeyedRabbitMQClient(
         this IHostApplicationBuilder builder,
         string name,
+        string? connectionName = null,
```

Comment:
This is a breaking change. You need to add a new method.

Reply:
fixed